examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -114,6 +114,7 @@
         "Test_TC_ACFREMON_2_1",
         "Disabled due to HEPAFilterMonitoring not being enabled in Matter.framework for now:",
         "Test_TC_HEPAFREMON_1_1",
-        "Test_TC_HEPAFREMON_2_1"
+        "Test_TC_HEPAFREMON_2_1",
+        "Test_TC_BINFO_2_2"
```

Comment:
HEPAFilterMonitoring has nothing to do with Test_TC_BINFO_2_2.  Please make sure the "comments" here match what's actually going on.

Reply:
@bzbarsky-apple  Made new PR: https://github.com/project-chip/connectedhomeip/pull/27960.